builder/virtualbox/ovf/config.go

```diff
@@ -103,14 +101,12 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 		if err != nil {
 			errs = packer.MultiErrorAppend(errs, fmt.Errorf("source_path is invalid: %s", err))
 		}
-		// file must exist now.
-		fileURL, _ := url.Parse(c.SourcePath)
-		if fileURL.Scheme == "file" {
-			if _, err := os.Stat(fileURL.Path); err != nil {
-				errs = packer.MultiErrorAppend(errs,
-					fmt.Errorf("source file needs to exist at time of config validation: %s", err))
-			}
+		_, err := common.FileExistsLocally(c.SourcePath)
```

Comment:
since the first parameter isn't used, I feel like this should be rewritten to something like

`func ValidateFileExistence(string) bool`

and just return true or false

So instead of saying "return a 2x2 result matrix and we'll decide what to do", we can just wrap all our business logic in a yes/no answer.

You could also have it return an error instead of a bool if you feel the error from stat is useful

Reply:
oh I like this.  you're completely right; as used right now I really want a simple yes or no answer.